rpc/block_svc_routines.c

```diff
@@ -3029,11 +3035,14 @@ block_create_cli_1_svc_st(blockCreateCli *blk, struct svc_req *rqstp)
   }
 
   GB_METAUPDATE_OR_GOTO(lock, glfs, blk->block_name, blk->volume,
-                        errCode, errMsg, exist, "SIZE: %zu\nENTRYCREATE: SUCCESS\n", blk->size);
+                        errCode, errMsg, exist,
+                        "SIZE: %zu\nRB_SIZE: %s\nENTRYCREATE: SUCCESS\n",
```

Comment:
@pranithk We are feeding RB_SIZE to metafile, should we show this as part of in the gluster-block info ?

Reply:
@pkalever Will it be relevant when dynamic ring buffer size patch comes? If yes, then we should, otherwise no IMHO